iOS_SDK/OneSignalSDK/OneSignalInAppMessages/Controller/OSMessagingController.m

```diff
@@ -165,6 +165,7 @@ + (OSMessagingController *)sharedInstance {
     return sharedInstance;
 }
 
+// TODO: Used by tests only? Come back and see if this is needed.
```

Comment:
Can this TODO be addressed in this PR?

Reply:
This method is used by tests only, I left this TODO as I am unsure if we will need to rely on this implementation as we migrate tests in the future, or if we can reset `OSMessagingController` state differently. I can clarify this TODO.